astyanax-cql/src/main/java/com/netflix/astyanax/cql/retrypolicies/ChangeConsistencyLevelRetryPolicy.java

```diff
@@ -112,6 +114,22 @@ public RetryDecision onUnavailable(Statement query, ConsistencyLevel cl,
 			boolean shouldRetry = retryOnAllConditions || retryOnUnavailable;
 			return checkRetry(query, cl, shouldRetry);
 		}
+
```

Comment:
Is this change needed as part of this PR?

Reply:
Yes, it provides backwards-compatibility for the previous version's cql retry policy by handling the new mandatory onRequestError() interface introduced by the upgrade to cassandra-driver-core 3.3.2.